### ISSUE TITLE
Bug: `hl-VertSplit` is deprecated.

### ISSUE THREAD

**Issue by pysan3** [metalelf0/jellybeans-nvim#7]
As title. `hl-VertSplit` is deprecated and has been renamed to `hl-WinSeparator`.

I'm not familiar with lush and do not know if _they_ are up to date, but it will be nice if you could define `hl-WinSeparator` or add something equivalent to `:hi default link WinSeparator VertSplit`.

https://neovim.io/doc/user/deprecated.html#_highlights

Thanks in advance.

**Comment by metalelf0** [comment]
@pysan3 Thanks for reporting this, should be fixed!

**Comment by pysan3** [comment]
Thank you very much for your quick response @metalelf0 !

I updated the plugin and now I get a _cannot access `injected_functions`_ error.

Turns out that lush does not have a new tag release since it introduced this feature and I only use the latest tag with `version = "*"` in lazy.nvim.

I think you are discussing about tree-sitter support with lush right now? and I see that the issue is still open? I can solve this with `version = false` on my end so it's not a big problem, but just wanted you to know.

Thanks for the amazing colorscheme tho!
